python-sdk/src/astro/lineage/extractor.py

```diff
@@ -39,9 +39,15 @@ def get_operator_classnames(cls) -> list[str]:
             "TransformOperator",
         ]
 
-    def extract(self) -> TaskMetadata | None:  # skipcq: PYL-R0201
+    def extract(self) -> TaskMetadata:  # skipcq: PYL-R0201
```

Comment:
not touched by this PR but instead of adding `skipcq` we should fix it by making it a `staticmethod`